assets/base.css

```diff
@@ -1057,6 +1057,26 @@ summary::-webkit-details-marker {
   }
 }
 
+@media screen and (max-width: 989px) {
+  .grid--1-col-tablet-down .grid__item {
+    width: 100%;
+    max-width: 100%;
+  }
+
+  .grid--2-col-tablet-down .grid__item {
+    width: calc(50% - var(--grid-desktop-horizontal-spacing) / 2);
+  }
+
+  .grid--4-col-tablet-down .grid__item {
```

Comment:
This ends up using `--grid-desktop-horizontal-spacing` for mobile as well. Far as I'm aware, tablet should still use the desktop spacing but mobile should use `--grid-mobile-horizontal-spacing`